Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/risklevel/ui/TestRiskLevelCalculationFragmentCWAViewModel.kt

```diff
@@ -129,26 +135,24 @@ class TestRiskLevelCalculationFragmentCWAViewModel @AssistedInject constructor(
         .toString()
 
     val additionalRiskCalcInfo = combine(
-        RiskLevelRepository.riskLevelScore,
-        RiskLevelRepository.riskLevelScoreLastSuccessfulCalculated,
-        exposureResultStore.matchedKeyCount,
-        exposureResultStore.daysSinceLastExposure,
+        riskLevelStorage.riskLevelResults,
         LocalData.lastTimeDiagnosisKeysFromServerFetchFlow()
```

Comment:
This also feels like something that could be part of riskLevelStorage?

Reply:
Hm, it's not risk level related though. Downloading and calculation is independent.
In any case, the actual timestamps are already tracked by `KeyPackageSyncSettings`, we could possibly remove the `LocalData.lastTimeDiagnosisKeys` soon, let's evaluate it after all PRs are merged for 1.8.x, what do you think?

Reply:
hmm true, maybe riskLevelStorage is the wrong place, reevaluation after the 1.8 merge sounds good to me :)